language/tools/move-cli/src/lib.rs

```diff
@@ -271,21 +269,7 @@ impl OnDiskStateView {
         Ok(())
     }
 
-    /// Save `resource` on disk under the path `addr`/`tag`
     pub fn save_resource(
-        &self,
```

Comment:
<3

Reply:
Interesting... I actually don't remember myself deleting this. Let me double check what's happening here.